cmd20.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+# Name: cmd20py
+# Authors: Luke Short | Joe R
+# Repo: https://github.com/ekultails/cmd20py/
+
+import random
+
+# Number Generator
+random.seed(None)
+die1 = random.randrange(0,7)
```

Comment:
We need to make this configurable by the user. This will also require a related test. By using a specific seed, our tests will be consistent. If given seed `foo`, with these numbers, then we expect `bar` to be returned.

Reply:
Could you expand on this a bit more? Are you suggesting that the user be able to set how many numbers the dice will have or rather generate? If so could you point me in the right direction on building configs?

Also should we use unittest library for creating the related test?

Reply:
For example, a user should be able to run `cmd20py --dice-sides 6 --roll-count 3 --seed 123456`. With that seed being passed to the random generator the results will always be the same. By default we should prefer generating a truly random seed.

Yes, we should also look into adding tests now. Allowing the seed to be specified will help us to have consistent results in our tests.